docs/superpowers/plans/2026-05-28-testing-foundation-stage3.md

```
@@ -0,0 +1,542 @@
+테스트에 하드코드할 값. 이미 실제 실행으로 확인됨:
+
+**Building blocks:**
+| BB | n_atoms | n_connection_points | is_edge | has bond block |
+|----|---------|---------------------|---------|----------------|
+| N10 | 12 | 3 | False | yes |
+| N409 | 18 | 4 | False | yes |
+| E41 | 8 | 2 | True | yes |
+| N198 | 28 | 6 | False | yes |
+
+**Topologies:**
+| topo | n_slots | n_nodes | n_edges | n_node_types | n_edge_types | spacegroup |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)